modules/mobile/lib/mobile/v0/vaos_appointments/appointments_helper.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require 'common/exceptions'
+
+module Mobile
+  module V0
+    module VAOSAppointments
+      class AppointmentsHelper
+        def initialize(user)
+          @current_user = user
+        end
+
+        def create_new_appointment(params)
+          get_appointment_location_info(get_new_appointment(params))
```

Comment:
I would recommend re-naming these methods and breaking them across lines. `get_new_appointment` isn't a get, it's a post, which confused me when reading it. I was thinking, "how are we actually creating the appointment?" Maybe something like:
```
def create_new_appointment(params)
  appointment = post_appointment(params)
  backfill_location(appointment)
end
```